README.md

```diff
@@ -1 +1,69 @@
-# tic_tac_toe_ruby
\ No newline at end of file
+# tic_tac_toe_ruby
```

Comment:
Readme was **optional** for this submission. However, it's great that you tried to implement it in this step! Kudos to you team! 🥳